dll/win32/kernel32/client/path.c

```diff
@@ -1114,6 +1115,26 @@ GetFullPathNameW(IN LPCWSTR lpFileName,
                                 lpFilePart) / sizeof(WCHAR);
 }
 
+/*
+ * @implemented
+ */
+BOOL
+WINAPI
+SetSearchPathMode(DWORD dwFlags)
```

Comment:
```suggestion
SetSearchPathMode(_In_ DWORD dwFlags)
```

Reply:
Is it necessary? In MSDN prototype, this parameter is not marked as input at all. Although technically it seems to be correct, but does it change anything?

Reply:
Except that there is an annotation when you look at the function prototype in MS PSDK' winbase.h.

Reply:
OK, then it seems to be indeed required.

Reply:
Committed, thanks.